events/static/events/js/script.js

```diff
@@ -0,0 +1,71 @@
+async function initMap() {
+    map = new google.maps.Map(document.getElementById("map"), {
+      center: { lat: 40.71159438471519, lng: -73.97424027294727 },
+      zoom: 12,
+      mapId: '9a7e26117f10196d'
```

Comment:
is this some kind of secret key for any account ? if yes we should add here this will get leaked

Reply:
Map Id doesn't really matter. But the Map API key is kind of a secret key, there's no way we can pass it through the script src file without leaking it from what I saw.